src/utils/helpers.ts

```diff
@@ -143,6 +144,38 @@ export const debounceAsyncValidator = <T>(validator: (value: T, callback: () =>
   };
 };
 
+// 900 -> 15 mins
+export const idleTimeTracker = ((idleTimeThreshold) => {
```

Comment:
where is this user for?

Reply:
to track if user is idle for a certain time and disable chain calls if so